imports/pages/groups/finder/Result.js

```diff
@@ -48,36 +49,38 @@ const mapStateToProps = ({ routing }) => {
   return { tags, q, location, campuses };
 };
 
-const mapQueriesToProps = ({ ownProps }) => ({
-  campusLocations: {
-    query: gql`
-      query GetCampuses { campuses { entityId, id, name } }
-    `,
-  },
-  data: {
-    query: gql`
-      query GroupFinder($query: String, $tags: [String], $limit: Int, $offset: Int, $ip: String, $campuses: [String]) {
-        groups(query: $query, attributes: $tags, limit: $limit, offset: $offset, clientIp: $ip, campuses: $campuses) {
-          count
-          results {
-            id
-            name
-            entityId
-            type
-            kidFriendly
-            demographic
-            description
-            photo
-            ageRange
-            distance
-            schedule { description }
-            locations { location { latitude, longitude } }
-            tags { id, value }
-            campus { name, entityId }
-          }
-        }
+const CAMPUS_LOCATION_QUERY = gql`
+  query GetCampuses { campuses { entityId, id, name } }
+`;
+
+const withCampusLocations = graphql(CAMPUS_LOCATION_QUERY, { name: "campusLocations" });
+
+const GROUP_FINDER_QUERY = gql`
+  query GroupFinder($query: String, $tags: [String], $limit: Int, $offset: Int, $ip: String, $campuses: [String]) {
+    groups(query: $query, attributes: $tags, limit: $limit, offset: $offset, clientIp: $ip, campuses: $campuses) {
+      count
+      results {
+        id
+        name
+        entityId
+        type
+        kidFriendly
+        demographic
+        description
+        photo
+        ageRange
+        distance
+        schedule { description }
+        locations { location { latitude, longitude } }
+        tags { id, value }
+        campus { name, entityId }
       }
-    `,
+    }
+  }
+`;
+
+const withGroupFinder = graphql(GROUP_FINDER_QUERY, {
+  options: (ownProps) => ({
```

Comment:
@edolyne this caused a lint failure. Can you change it to `options: ownProps => ({`